cfme/common/vm_views.py

```diff
@@ -372,13 +372,13 @@ class ProvisionView(BaseLoggedInPage):
     """
     title = Text('#explorer_title_text')
     breadcrumb = BreadCrumb()
+    image_table = Table('//div[@id="pre_prov_div"]//table')
```

Comment:
Because it is a table which is present before the form and its inheritances( BasicProvisionFormView and Tab). So Before_fill was not able to find elements in the table. and the table is present only for provisioning  ( not while cloning/migrating)

Reply:
REQUIRED: sounds good, but there is code which looks for this widget in form. So, please check all dependencies and fix them before such change
cloud->test_providers.py:
```
    view = navigate_to(Instance, 'Provision')
    view.form.image_table[0].click()
```